src/supplemental_accessors.jl

```diff
@@ -1,16 +1,19 @@
-"""Get Forecast component name."""
-get_forecast_component_name(forecast::Forecast) = get_name(get_component(forecast))
+"""
+    get_forecast_value(val::Deterministic, ix)
 
-"""gets the value of a Deterministic forecast at a given index or DateTime timestamp"""
+Return the value of a Deterministic forecast at a given index or DateTime timestamp.
+"""
```

Comment:
I think this is no longer needed given the implementation of get_forecast_value in line 130 of Forecasts.jl

Reply:
You can call this function if you have an instance of Forecast.  The function on line 130 of component.jl allows you to call it without an instance of a forecast.  Both seem useful.  Are you sure you want to remove this one?

```
130 function get_forecast_value(
131                             ::Type{T},
132                             component::InfrastructureSystemsType,
133                             initial_time::Dates.DateTime,
134                             label::AbstractString,
135                             index::Union{Int, Dates.DateTime},
136                            ) where T <: ForecastInternal
```

Reply:
ok